src/server/scripts/Northrend/Naxxramas/boss_grobbulus.cpp

```diff
@@ -15,35 +15,51 @@
  * with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
+/*
+ * Slime Spray timers must always looks like ~30 (initial) ~30 (first repeat) 55 (second repeat) ~30 55 ~30 55 ~30 55 ~30 55
+  so after second cast timers are always flipped
+ * SPELL_BOMBARD_SLIME should be used out of combat (waypoint script?)
+ * He should call all stitched giants after aggro
+ */
```

Comment:
usually these type of comments are put near the thing it references, not a the top

Reply:
That's kinda more correct version of old `SDComment:`.  Right now I don't have a clear vision of how it's supposed to look. I still add some comments inside code but I think about something like 'global TO-DO per file'

All that stuff wasn't bad but always looked to me useless. I mean, one line for comments usually filled with comments like 'timers, something is broken'. TC mostly removes all old ScriptData but we still need some global TO-DO per file
```cpp
/* ScriptData
SDName: Boss_High_King_Maulgar
SD%Complete: 90
SDComment: Correct timers, after whirlwind melee attack bug, prayer of healing
SDCategory: Gruul's Lair
EndScriptData */
```

Not sure about slime timers but comment about SPELL_BOMBARD_SLIME can be moved to enum
And part about call for help cannot be moved anywhere